Integrations/src/test/java/io/deephaven/integrations/learn/InputTest.java

```diff
@@ -0,0 +1,26 @@
+package io.deephaven.integrations.learn;
+
+import junit.framework.TestCase;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.PrimitiveIterator;
+
+import io.deephaven.integrations.*;
+
+/*
```

Comment:
Not tested.  Better to have a test skeleton with a fail test, so that you don't forget to write the test.